PendingReleaseNotes

```diff
@@ -139,6 +139,24 @@
 * CEPHFS: After recovering a Ceph File System post following the disaster recovery
   procedure, the recovered files under `lost+found` directory can now be deleted.
 * core: cache-tiering is now deprecated.
+* mClock Scheduler: The mClock scheduler (introduced in Quincy) has undergone
+  significant changes both user visible and under the hood to address the slow
+  backfill issue. Some important changes are:
+  * The 'balanced' profile is set as the default mClock profile.
```

Comment:
It is worth mentioning the reason for this change.

Reply:
Changed.